### PR TITLE
BVAL-616 Adding @PositiveOrZero and @NegativeOrZero constraints

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

**Comment by gsmet** [comment]
@gunnarmorling you forgot to update sources/appendix-standardresolvermessages.asciidoc .

**Comment by gunnarmorling** [comment]
Pushed one more commit for adding/updating the default messages.

File: sources/appendix-standardresolvermessages.asciidoc [hunk 1]
```diff
@@ -35,6 +37,8 @@ javax.validation.constraints.Past.message=must be a past date
 javax.validation.constraints.Pattern.message=\
     must match the following regular expression: {regexp}
 javax.validation.constraints.Positive.message=\
-    must be greater than ${strict == true ? '' : 'or equal to '}0
+    must be greater than 0
+javax.validation.constraints.PositiveOrZero.message=\
+    must be greater than or or equal to 0
```
Comment (gsmet) [inline sources/appendix-standardresolvermessages.asciidoc]: 2 `or` here. Will fix and merge.

**Comment by gunnarmorling** [comment]
Agh, good catch. Thanks!

**Comment by gsmet** [comment]
Force pushed a fix.

Will merge this one once we have the RI ready. Otherwise it will break CI.

**Comment by gunnarmorling** [comment]
@gsmet 

> Force pushed a fix.

Just saw this. I think I've overwritten this by rebasing and force-pushing myself, sorry. Can you apply your fix again?

**Pull request merged**